### PR TITLE
`SiPixelDynamicInefficiency_PayloadInspector`: add region descriptor to title of inefficiency parametrization plots 

### PR DESCRIPTION
**Pull request by mmusich**
#### PR description:

This is an immediate follow-up to https://github.com/cms-sw/cmssw/pull/40856 in order to add region descriptor to title of inefficiency parametrization plots introduced there (for ease of interpreting the results).
As a consequence, I've also found out a bug in the determination of inner / outer ladders in `SiPixelPI::topolInfo` which I correct in commit 3a96205c715998435bb018f53c75faf8234b4b34.

#### PR validation:

Relies on the existing unit tests introduced at:https://github.com/cms-sw/cmssw/pull/40856 .
Also run privately the following command 
```
getPayloadData.py \
    --plugin pluginSiPixelDynamicInefficiency_PayloadInspector \
    --plot plot_SiPixelDynamicInefficiencyPUParametrization \
    --tag SiPixelDynamicInefficiency_PhaseI_v9 \
    --time_type Run \
    --iovs '{"start_iov": "1", "end_iov": "1"}' \
    --db Prod \
    --test;
```
and obtained the following plot:

image
 

#### If this PR is a backport please specify the original PR and why you need to backport that PR. If this PR will be backported please specify to which release cycle the backport is meant for:

N/A

### CODE REVIEW

**Comment by mmusich** [comment]
type bug-fix

File: CondCore/SiPixelPlugins/plugins/SiPixelDynamicInefficiency_PayloadInspector.cc [hunk 1]
```diff
@@ -771,16 +771,42 @@ namespace {
...
+          namesOfParts.push_back(regName);
+          /*
+	    // debug
+	    edm::LogVerbatim(label_) << "region: " << index << " " << regName << " has the following modules: ";
+	    for (const auto& module : modules) {
+              emd::LogVerbatim(label_) << module << " ";
+	    }
+	    edm::LogVerbatim(label_) << std::endl;
+	  */
+        }
```
Comment (tvami) [inline CondCore/SiPixelPlugins/plugins/SiPixelDynamicInefficiency_PayloadInspector.cc:808]: I guess this should be removed now.

Comment (mmusich) [reply CondCore/SiPixelPlugins/plugins/SiPixelDynamicInefficiency_PayloadInspector.cc]: removed.

**Comment by mmusich** [comment]
@cmsbuild, please test

**Comment by mmusich** [comment]
[unit tests passed](https://cmssdt.cern.ch/SDT/jenkins-artifacts/pull-request-integration/PR-0fb733/30869/unitTests/src/CondCore/SiPixelPlugins/test/testPixelPayloadInspector/testing.log)

**Comment by francescobrivio** [comment]
+1

**Pull request merged**